firefox-ios/Client/TabManagement/TabManager.swift

```diff
@@ -15,8 +15,9 @@ enum TabsDeletionPeriod: String {
 }
 
 // MARK: - TabManager protocol
+@MainActor
 protocol TabManager: AnyObject {
-    var windowUUID: WindowUUID { get }
+    nonisolated var windowUUID: WindowUUID { get }
```

Comment:
There is no other choice than marking the `windowUUID` nonisolated? Is this something to improve with a follow up ticket?